prow/plugins/trigger/generic-comment.go

```diff
@@ -43,12 +45,35 @@ func handleGenericComment(c Client, trigger plugins.Trigger, gc github.GenericCo
 	// Skip comments not germane to this plugin
 	if !pjutil.RetestRe.MatchString(gc.Body) && !pjutil.OkToTestRe.MatchString(gc.Body) && !pjutil.TestAllRe.MatchString(gc.Body) {
```

Comment:
The validation you added assumes that only one command is specified. The following comment wouldn't trigger an error.
```
/retest
/test non-existent-job
```

Reply:
I didn't add this validation. It's from the original code.

Reply:
There was no existing validation... That is why `/test non-existent-job` didn't return an error.
The code you added validates that the command is sensible, but it incorrectly assumes that there is only one command per comment.